downloadDirectory/src/main/java/org/reactome/release/downloadDirectory/CreateReleaseTarball.java

```
@@ -0,0 +1,99 @@
+import org.eclipse.jgit.api.errors.GitAPIException;
+
+public class CreateReleaseTarball {
+	private static final Logger logger = LogManager.getLogger();
+	public static boolean runPerl = true;
+	
+	public static void execute(String releaseNumber, String releaseDownloadDir) throws IOException, InterruptedException {
+		logger.info("Running CreateReleaseTarball step");
+		
+		if (runPerl) {
+			String tarballCommand = "perl " + releaseDownloadDir + "/make_release_tarball.pl " + releaseNumber;
+			Process runPerlTarballCommand = Runtime.getRuntime().exec(tarballCommand);
+			runPerlTarballCommand.waitFor();
+		} else {
+		
+		
+			String tarDirRelease = "reactome_tar/" + releaseNumber + "/Release";
+			String absReleaseDir = "/usr/local/gkb/scripts/release/website_files_update";
+			String absReactomeDir = "/usr/local/reactomes/Reactome/production/Website/static/download/" + releaseNumber;
+			releaseNumber = releaseNumber.replace("/", "");
+			//TODO: When the dust has settled, make sure the tarball contains everything we need
+			
+			// Remove any existing Release repositorys
+			File releaseDir = new File(tarDirRelease);
+			releaseDir.mkdirs();
+			if (releaseDir.list().length > 0) {
+				Process removeReleaseRepo = Runtime.getRuntime().exec("rm -r " + tarDirRelease);
+				removeReleaseRepo.waitFor();
+			} 
+			logger.info("Cloning Release: https://github.com/reactome/Release.git...");
+			try {
+				Git.cloneRepository().setURI("https://github.com/reactome/Release.git").setDirectory(Paths.get(tarDirRelease).toFile()).call();
+				logger.info("Finshed cloning Release repository");
+			} catch (GitAPIException e) {
+				e.printStackTrace();
+			} 
+			logger.info("Copying files into reactome_tar directory...");
+			String tarDirGKB = "reactome_tar/" + releaseNumber + "/reactome/GKB";
+			new File(tarDirGKB).mkdirs();
+			try {
+				FileUtils.copyDirectory(new File(tarDirRelease + "/website"), new File(tarDirGKB));
+			} catch (FileNotFoundException ignore) {}
```

Could a log statement be added to the catch?  It's best practice not to ignore the exception, but if it's really okay then a comment explaining why it's ignored would be great.

---

Yea I'm not sure why this is here like that. It's possible I was testing something. For the last release the tarball step just used the Perl code, so maybe I forgot to clean this up